conformance.adoc

```diff
@@ -552,6 +552,8 @@ a geometry container variable. The variable name must exist in the file.
 **`node_coordinates`** attributes.
 * The only legal values of geometry_type are **`point`**, **`line`**,
 and **`polygon`** (case insensitive).
+* For a line **`geometry_type`**, each geometry must have a minimum of two node coordinates.
+* For a polygon **`geometry_type`**, each geometry must have a minimum of three node coordinates.
```

Comment:
Shouldn't "three" be "two", in which case, we could merge two rules into one:

  * For a line or polygon `geometry_type`, each geometry must have a minimum of two node coordinates.

E.g. In Example 7.16. (Polygons with holes), we have only two node coordinates. (https://github.com/twhiteaker/cf-conventions/blob/master/ch07.adoc)

Reply:
I was referring to the size of the node dimension, not the number of node coordinate variables.  Is this rule clearer?

* For a line **`geometry_type`**, each geometry must have a minimum of two nodes.